ateam_ssl_vision_bridge/src/message_conversions.cpp

```diff
@@ -28,14 +28,16 @@
 namespace ateam_ssl_vision_bridge::message_conversions
 {
 
+constexpr double mmTom = 1.0 / 1000;
```

Comment:
nit / alternative: You can use scientific notation in C++.
```suggestion
constexpr double mmTom = 1.0e-3;
```